keyboards/momokai/kei_6/readme.md

```diff
@@ -0,0 +1,13 @@
+# Momokai Kei 6
+
+A 6 key keypad with a rotary encoder
+
+* Keyboard Maintainer: [peepeetee](https://github.com/peepeetee)
+
```

Comment:
```suggestion
* Keyboard Maintainer: [peepeetee](https://github.com/peepeetee)
* Hardware Supported: Kei 6
* Hardware Availability: [Momokai](https://www.momokai.com/)
```

Is hardware information and availability for this and other momokai product excluded for reasons? Otherwise this information is prudent for users to accurately match firmware to hardware.

Reply:
Resolving with no response doesn't help anyone.

Reply:
I resolved this since I um committed the suggestion?

the availability on the website should be updated later after they enter production